metadata-transformer/src/main/java/no/unit/nva/metadata/service/MetadataService.java

```diff
@@ -88,18 +102,33 @@ public MetadataService(HttpClient httpClient) {
         }
     }
 
+    public Optional<String> lookUpJournalIdAtPublicationChannel(String name,
```

Comment:
The naming of the methods is a bit problematic, but I think this needs to be a new PR as it requires multiple changes — create a refactoring task for that.

Reply:
Where do you want that refactoring task? Smile board?
Do you have some details what you are aiming at to be improved?